test/ParseableInterface/ModuleCache/SDKDependencies.swift

```diff
@@ -20,8 +38,12 @@
 // RUN: test -f %t/MCP/SdkLib-*.swiftmodule
 //
 // Check they are *not* forwarding modules
-// RUN: head -n 1 %t/MCP/SdkLib-*.swiftmodule | not grep -e '---'
-// RUN: head -n 1 %t/MCP/ExportedLib-*.swiftmodule | not grep -e '---'
+// RUN: not python %S/Inputs/check-is-forwarding-module.py %t/MCP/SdkLib-*.swiftmodule
+// RUN: not python %S/Inputs/check-is-forwarding-module.py %t/MCP/ExportedLib-*.swiftmodule
```

Comment:
Nitpick: `%{python}` ensures that we use the same Python being used to run lit.py, which can be important on systems with Python 3 installed.